doc/nrf/ug_matter_gs_testing.rst

```diff
@@ -9,7 +9,7 @@ When you build any of the available Matter samples to the supported development
 The development kit and the application running Matter stack that is programmed on the development kit together form the Matter accessory device.
 
 The |NCS| supports Matter stack that is built on top of a low-power, 802.15.4-compatible Thread network, or on top of a Wi-Fi network.
-To control the Matter accessory device remotely over either of these networks, you need to set up a Matter controller on a mobile or a PC.
```

Comment:
I'm not sure about removing such information, as now it looks a little bit, like mobile support would be dropped at all (which is not true, because user still can use mobile controllers from ecosystems). But it's ok, we can handle it btw of adding ecosystems page in doc.

Reply:
Yeah, I thought about it that way we can remove it because we will have an ecosystem page, but hmm, maybe it is worth adding here info about mobile controllers from ecosystems instead of removing that sentence.